### PR TITLE
zh-translation: docs/examples/microservices-istio/single/index.md

### PR DESCRIPTION
**Pull request by homilly**
ref:  [servicemesher/istio-official-translation#1230](https://github.com/servicemesher/istio-official-translation/issues/1230)

### CODE REVIEW

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 1]
```diff
@@ -53,24 +41,21 @@ and run it locally:
...
-    - The web server's features:
-        - listening to a port
-        - handling requests and responses
-    - The aspects related to HTTP:
+1. 浏览服务的代码，并注意代码中以下元素：
+    - web 服务器的特征:
+        - 监听一个端口
+        - 处理请求和响应
+    - 与 HTTP 相关的方面:
         - headers
```
Comment (ilylia) [inline content/zh/docs/examples/microservices-istio/single/index.md]: 请求头

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 2]
```diff
@@ -53,24 +41,21 @@ and run it locally:
...
-        - path
-        - status code
+        - 路径
+        - status code 状态码
```
Comment (ilylia) [inline content/zh/docs/examples/microservices-istio/single/index.md]: 删除英文

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 3]
```diff
@@ -53,24 +41,21 @@ and run it locally:
...
-    web application runs as a standalone process.
+    在 Node.js 中, web服务器的功能嵌入在应用程序的代码中。 一个 Node.js
+    web 应用程序作为一个独立进程运行。
     {{< /tip >}}
 
-1. Node.js applications are written in JavaScript, which means that there is no
-    explicit compilation step. Instead, they use [just-in-time compilation](https://en.wikipedia.org/wiki/Just-in-time_compilation). To build a Node.js application, then means to install its dependencies. Install
-    the dependencies of the `ratings` service in the same folder where you
-    stored the service code and the package file:
+1. Node.js 应用程序是用 JavaScript 编写的， 这意味着没有显式编译步骤。 相反，它们使用 [just-in-time 即时编译](https://en.wikipedia.org/wiki/Just-in-time_compilation)。 要构建 Node.js 应用程序， 则意味着要安装其依赖库。 将 `rating` 服务的依赖库安装在存储服务代码和 package 文件的同一目录下：
```
Comment (ilylia) [inline content/zh/docs/examples/microservices-istio/single/index.md]: https://en.wikipedia.org/wiki/Just-in-time_compilation
=>
https://zh.wikipedia.org/wiki/%E5%8D%B3%E6%97%B6%E7%BC%96%E8%AF%91

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 4]
```diff
@@ -92,38 +77,36 @@ and run it locally:
...
 {{< tip >}}
-The `ratings` service is a web app and you can communicate with it as you would
-with any other web app. You can use a browser or a command line web client like
-[`curl`](https://curl.haxx.se) or [`Wget`](https://www.gnu.org/software/wget/).
-Since you run the `ratings` service locally, you can also access it via the
-`localhost` hostname.
+该 `ratings` 服务是一个 web 应用程序，您可以像访问其他 web 应用程序那样访问它。
+ 您可以使用浏览器或者像
+ [`curl`](https://curl.haxx.se) 或 [`Wget`](https://www.gnu.org/software/wget/)
+的 命令行 web 客户端。由于您在本地运行了 `rating` 服务，因此您也可以通过 `localhost`
```
Comment (ilylia) [inline content/zh/docs/examples/microservices-istio/single/index.md]: 删除多余空格

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 5]
```diff
@@ -92,38 +77,36 @@ and run it locally:
...
+1. 在浏览器中打开 [http://localhost:9080/ratings/7](http://localhost:9080/ratings/7) 或者使用 `curl` 命令来访问 `ratings`：
 
     {{< text bash >}}
     $ curl localhost:9080/ratings/7
     {"id":7,"ratings":{"Reviewer1":5,"Reviewer2":4}}
     {{< /text >}}
 
-1. Use the `POST` method of the `curl` command to set the ratings for the
-    product to `1`:
+1. 使用 curl 命令的 POST 方法将产品的等级设置为1：
```
Comment (ilylia) [inline content/zh/docs/examples/microservices-istio/single/index.md]: 等级 => 评级
为1 => 为 `1`

**Comment by homilly** [comment]
@googlebot I signed it!

**Pull request merged**